velox/exec/Window.h

```diff
@@ -22,6 +22,16 @@
 
 namespace facebook::velox::exec {
 
+// Structure for k preceding/following frame arguments, constantValue stores
+// the value of k for constant k and the column index when k is a column. For
+// columnar values of k, the columnarOffset stores the offsets for the rows
+// being parsed in the call to 'apply'.
+struct FrameArg {
+  bool isConstantValue;
```

Comment:
I feel its better to have 2 fields : (optional) constantValue and constantIndexValue (this can be kConstantChannel) instead of the bool. I didn't follow the explanation for columnOffset-- this is the vector of values from the frame channel ?

Reply:
Thanks for the suggestion, I will use these 2 fields and remove the bool. columnOffset is a FlatVector that stores the value in the column at the given index and is populated for numRows in the call to apply().